### DISCUSSION TITLE
How to install it on Jetson Nano?

### DISCUSSION THREAD

**Discussion by shrinand1996** [open-edge-platform/anomalib#604] [Q&A]
I would like to install it on a jetson nano but I am unable to do so. The installation works fine in a conda environment with some changes but the GPU is not utilized. I figured this was due to the python 3.8 requirement for pytorch. 

I can install pytorch 1.9 on python 3.6. Jetson does not support python3.8. 

How can i use this library on the jetson? 

To use the model and test it out, I need to deploy it first on an edge device.

**Comment by ashwinvaidya17** [comment]
@shrinand1996 What's the cuda version when you run `nvidia-smi`? Did you by chance install cpu only version of pytorch? What's the output of this?
```python
import torch
print(torch.cuda.is_available())
```
